airflow/script/entrypoint.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+CMD="airflow"
+TRY_LOOP="10"
+POSTGRES_HOST="postgres"
+POSTGRES_PORT="5432"
+RABBITMQ_HOST="rabbitmq"
+RABBITMQ_CREDS="airflow:airflow"
+FERNET_KEY=$(python -c "from cryptography.fernet import Fernet; FERNET_KEY = Fernet.generate_key().decode(); print FERNET_KEY")
+
+# Generate Fernet key
+sed -i "s/{FERNET_KEY}/${FERNET_KEY}/" $AIRFLOW_HOME/airflow.cfg
+
+# wait for rabbitmq
+if [ "$1" = "webserver" ] || [ "$1" = "worker" ] || [ "$1" = "scheduler" ] || [ "$1" = "flower" ] ; then
```

Comment:
unless i'm mistaken -- you should be able to remove these lines and instead leverage a kubernetes entrypoint to wait until postgres and rabbit have endpoints within kubernetes